install/index.md

```diff
@@ -48,6 +48,7 @@ Use the selector tool below to select your preferred method, packages, and envir
 ## Installation Troubleshooting
 
 ### **Conda Issues**
+- Ensure that the `defaults` channel is not included in the conda solve. The `defaults` channel is not compatible with `conda-forge`, which RAPIDS relies on. If you installed conda with [the Miniconda or Anaconda distributions](https://www.anaconda.com/docs/getting-started/miniconda/main#should-i-install-miniconda-or-anaconda-distribution), the `defaults` channel will be included unless you modify your `.condarc` file or specify `nodefaults` in the install commands for RAPIDS packages. If you installed with [Miniforge](https://conda-forge.org/download/) (our recommendation for best compatibility) then the `defaults` channel is not included. Note that in general [mixing `conda-forge` and `defaults` packages is not supported by either](https://conda-forge.org/docs/user/transitioning_from_defaults/).
```

Comment:
The content is good but the formatting needs to fit into the page. The hints below have an info bubble ℹ️ with a problem (e.g. "A `conda create` error occurs") followed by possible solutions.

What are the problems that users will observe if they accidentally mix the channels? You can preview the rendering at https://deploy-preview-624--docs-rapids-ai.netlify.app/install/.

Reply:
This is reasonable info and agree with Bradley's suggestion

Would also recommend starting this section by linking to our [Conda install docs]( https://docs.rapids.ai/install/#conda ) and pointing out the installer and channel we recommend

Then move to stating `defaults` incompatibilities and how the `defaults` channel might be coming into play

Reply:
Done. Didn't start with our docs since that doesn't really fit the problem->solution structure so well here but I did link them.